fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/TransactionManagerImplTest.java

```diff
@@ -60,6 +60,12 @@ public void testGetTransaction() {
 
     @Test(expected = RuntimeException.class)
     public void testGetTransactionWithInvalidID() {
-        final TransactionImpl tx = (TransactionImpl) testTxManager.get("invalid-id");
+        testTxManager.get("invalid-id");
```

Comment:
Lets at least get this test expecting a RepositoryRuntimeException instead of RuntimeException. Might have to change the class for that.